### DISCUSSION TITLE
Chrome browser extension just disappeared

### DISCUSSION THREAD

**Discussion by sumeetsingh39** [subhra74/xdm#898] [Q&A] [answered]
Chrome extension just disappeared recently from my system, and the URL to install the extension doesn't work.

**Comment by Demabanksy** [comment, marked as answer]
same here

**Reply by riseoff31131** [reply]
https://chrome.google.com/webstore/detail/xdm-integration-module-be/akdmdglbephckgfmdffcdebnpjgamofc/related

**Reply by riseoff31131** [reply]
https://github.com/subhra74/xdm/discussions/768

**Reply by mj192000** [reply]
> https://chrome.google.com/webstore/detail/xdm-integration-module-be/akdmdglbephckgfmdffcdebnpjgamofc/related

on using this extension its showing that register extension with xdm how do i do that

**Reply by dumbfuckjuic** [reply]
i think that one is fake use another way like using github

**Reply by sh142kimo** [reply]
有辦法使用嗎

**Comment by dongdadongdondga** [comment]
+1

**Comment by bcooly** [comment]
Yes same issue, chrome extension is disappeared also in vivaldi extension

**Comment by andewrisc** [comment]
if u use chrome the extension in old version is doesnt exist. you can install with developer mode in chrome follow the intructions https://sourceforge.net/p/xdman/blog/2018/01/xdm-integration-with-microsoft-edge/

**Reply by andewrisc** [reply]
1. extract the file
2. turn-on developer extension -> load pack -> chose the file
3. DONE

**Reply by RouryR** [reply]
after reboot my pc the extension was deleted

**Reply by dumbfuckjuic** [reply]
stop using sourceforgenet lol

**Reply by bcooly** [reply]
Why?

**Comment by durhambell** [comment]
Same. It was automatically disabled by Edge. In my case I got an error when I tried to reenable it. Apparently Microsoft has flagged it as malware?
![XDM Screenshot 2022-10-01 173557](https://user-images.githubusercontent.com/9648408/193429266-ec4c5c38-def7-4129-9f7b-7691f36fec2e.png)

Edge will not let me reenable it. Possibly because it is sideloaded?

**Comment by subhra74** [comment]
For those who are having issues with browser extension, please install latest version from https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.11-beta.
The chrome extension must be installed from XDM browser monitoring screen. Instructions to install the extension is provided there as well.

**Reply by sh142kimo** [reply]
好像只有YU 可以用而已    
https://pttplay.cc/v/219544-1-1.html 
這個無法使用

**Reply by olaem71** [reply]
It is better but video grabber only works on youtube videos. I do not understand, this was working perfectly before, why this issues.

**Reply by olaem71** [reply]
https://gogoanimeapp.com/

**Reply by Gasiyu** [reply]
is it possible to make the rpm package depends on ffmpeg (from rpmfusion) instead of ffmpeg-free?

**Comment by coucoseth** [comment]
links in marked answer are not working, any solution
